postgres/client.sls

```diff
@@ -25,23 +22,23 @@ postgresql-client-libs:
       - pkgrepo: postgresql-repo
 {%- endif %}
 
+# Alternatives system. Make client binaries available in $PATH
 {%- if 'bin_dir' in postgres %}
+  {% if postgres.linux.altpriority|int > 0 %}
```

Comment:
At first, you don't need to introduce lower level `if` statement, it's better to collapse two statements into one simply using `and` operator. It reduces amount of changes, unnecessary indentations and makes it more readable overall.

At second, no need to use `int` filter here. You could just test with implicit Boolean comparison:
```jinja
  {%- if 'bin_dir' in postgres and
          postgres.linux.altpriority %}
```
This would negate the check if `postgres.linux.altpriority` would be `0`, `''`,`false` or `none`. And that looks better.

Reply:
Nice suggestion.  This is all done.